### ISSUE TITLE
make error with external libs

### ISSUE THREAD

**Issue by RekGRpth** [michaelrsweet/htmldoc#411]
make error with external libs after commit https://github.com/michaelrsweet/htmldoc/commit/1456c4fcd8971161dc71c1d0ba17150f0a558ba3
```sh
c++: error: ../png/libpng.a: No such file or directory
c++: error: ../zlib/libz.a: No such file or directory
c++: error: ../jpeg/libjpeg.a: No such file or directory
```
solve
```diff
diff --git a/configure b/configure
index a338aa4..94fe552 100755
--- a/configure
+++ b/configure
@@ -6183,7 +6183,8 @@ then :
   test "$ac_res" = "none required" || LIBS="$ac_res $LIBS"
 
        JPEGINC=""
-       JPEG="",
+       JPEG=""
+else
        JPEGINC="-I../jpeg"
        JPEG="jpeg"
        NEWLIBS="../jpeg/libjpeg.a $NEWLIBS"
@@ -6270,7 +6271,8 @@ then :
   test "$ac_res" = "none required" || LIBS="$ac_res $LIBS"
 
        ZLIBINC=""
-       ZLIB="",
+       ZLIB=""
+else
        ZLIBINC="-I../zlib"
        ZLIB="zlib"
        NEWLIBS="../zlib/libz.a $NEWLIBS"
@@ -6357,7 +6359,8 @@ then :
   test "$ac_res" = "none required" || LIBS="$ac_res $LIBS"
 
        PNGINC=""
-       PNG="",
+       PNG=""
+else
        PNGINC="-I../png"
        PNG="png"
        NEWLIBS="../png/libpng.a $NEWLIBS"
```

**Comment by michaelrsweet** [comment]
@RekGRpth Actually I need to stick in some "], [" to get the else in there - that was all for migrating to the recommended autoconf stuff now that 2.70 is out...

**Comment by michaelrsweet** [comment]
[master 6898d0a] Fix regression in configure script (Issue #411)
